gammapy/image/utils.py

```diff
@@ -799,7 +833,24 @@ def make_header(nxpix=100, nypix=100, binsz=0.1, xref=0, yref=0,
 
     Parameters
     ----------
-    TODO
+    nxpix : int
+	Number of pixels in x axis. Default is 100.
```

Comment:
@adonath is also doing this ... repeating the default values in the docstring like @mapazarr you do here.

I see that it's convenient for the user, because otherwise they have to look at the function signature to see the default.

In Astropy some people do this (see e.g. http://astropy.readthedocs.org/en/latest/api/astropy.table.join.html#astropy.table.join) and some don't.

In numpy / scipy usually (always?) default values are not repeated in the docstring:
https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize.html

I would prefer we don't duplicate the default value in the docstring ([DRY](http://en.wikipedia.org/wiki/Don%27t_repeat_yourself)), because it's hard to make sure it's always consistent, i.e. not forget to update the docstring when the default changes.

@adonath - OK? Or do you feel strongly repeating the default values in the docstring is the better option?